README.md

```diff
@@ -83,6 +83,24 @@ iex> KafkaEx.create_worker(:pr, [uris: uris, consumer_group: "kafka_ex", consume
 {:ok, #PID<0.172.0>}
 ```
 
+### Create an unnamed KafkaEx worker
+
+You may find you want to create many workers, say in conjunction with
+a `poolboy` pool. In this scenario you usually won't want to name these worker processes.
+
+To create an unnamed worked with `create_worker`:
+```elixir
+iex> KafkaEx.create_worker(:no_name) # indicates to the server process not to name the process
+{:ok, #PID<0.171.0>}
+```
+
+To create an unnamed worker directly with the `KafkaEx.Supervisor`:
+```elixir
```

Comment:
That's fine, if you have a working example with poolboy I will appreciate if you can PR that separately in an examples folder.

Thanks!